intro-Linux/ch_getting_started.tex

```diff
@@ -94,13 +93,23 @@ \section{Variables}
 \subsection{Defining variables}
 
 There are several variables already defined for you when you start your
-session, such as ``\$HOME'' which contains the path to your home directory. 
+session, such as ``\$HOME'' which contains the path to your home directory.
+
+For a full overview of defined environment variables in your current session,
+you can use the ``env'' command:
+
+\begin{prompt}
+%\shellcmd{env}%
+...
+HOME=%\homedir%
```

Comment:
The `%homedir%` is going to confuse people, since that's not what they will be seeing.

Use the actual value, but anonimise it, like `vsc40000`. We probably need a macro for an anonimised VSC login in, since it'll be different for each site (`vsc4xxxx` for Ghent, etc.)

Reply:
`%\homedir%` is a macro, and gets expanded (? not sure if this is the right word) to `/user/home/gent/vsc400/vsc4000`.

Reply:
I overlooked that, my bad... If it gets expanded as expected in the rendered document, then it's OK.